tests/Middleware/RequestBodyBufferMiddlewareTest.php

```diff
@@ -93,6 +93,33 @@ function (ServerRequestInterface $request) {
         $this->assertSame('', $response->getBody()->getContents());
     }
 
+    public function testKnownExcessiveSizedWithIniLikeSize()
+    {
+        $loop = Factory::create();
+
+        $stream = new ThroughStream();
+        $loop->addTimer(0.001, function () use ($stream) {
+            $stream->end(str_repeat('a', 2048));
+        });
+        $serverRequest = new ServerRequest(
+            'GET',
+            'https://example.com/',
+            array(),
+            new HttpBodyStream($stream, 2)
```

Comment:
Does the 2 make sense here?

Reply:
I agree, sementically no, doesn't make sense. Practically doesn't make a difference though.

Reply:
Fixed it anyway 👍